src/components/UserListItem.js

```diff
@@ -1,31 +0,0 @@
-import React, { Component } from 'react'
```

Comment:
Unused component. Burce & Robbie created this component, but never used it. Not sure what it shows up again here...